tests/Feature/Api/ApiKeyManagerTest.php

```
@@ -118,6 +118,30 @@
+    // generation must therefore happen in-process, never through the console layer. See #167.
+    $emptyDir = storage_path('framework/testing/passport-keys-'.uniqid());
+    File::ensureDirectoryExists($emptyDir);
+    Passport::loadKeysFrom($emptyDir);
+    config(['passport.auto_generate_keys' => true, 'passport.private_key' => null, 'passport.public_key' => null]);
+
+    try {
+        Artisan::spy();
+
+        [$apiKey] = $this->manager->issue($this->workspace, $this->user, 'octane', 'read', null);
+
+        Artisan::shouldNotHaveReceived('call');
```

PHPStan (2.2.7)

[warning] Call to an undefined static method Illuminate\Support\Facades\Artisan::shouldNotHaveReceived().


(staticMethod.notFound)